ios/Exponent/Kernel/Services/EXUtilService.m

```diff
@@ -1,23 +0,0 @@
-// Copyright 2015-present 650 Industries. All rights reserved.
-
-#import "EXUtilService.h"
-#import "EXViewController.h"
-#import "ExpoKit.h"
-
-#import <ExpoModulesCore/EXDefines.h>
-
-@implementation EXUtilService
-
-EX_REGISTER_SINGLETON_MODULE(Util)
```

Comment:
Like in https://github.com/expo/expo/pull/24318#pullrequestreview-1618683640, this macro also has some side-effects in runtime, even if not referenced. Afaik this module is still used by the splash screen (look for `getSingletonModuleForName:@"Util"` method calls)